deploy/prepare.sh

```diff
@@ -37,6 +37,10 @@ fi
 echo "Ensuring kubespray is present"
 git submodule --quiet init
 git submodule --quiet update
+cd kubespray
+#check out to support kube 1.13.1
+git checkout 39d750
```

Comment:
This shouldn't be required. The `git submodule (init|update)` commands above should checkout the correct git hash  as you've also submodule git hash.

Reply:
@kshlm @Madhu-1 I this is overriding the submodule commit... I think this needs to be removed.
The submodule commands may also not be what we want... not really sure. We need to make sure that a commit of gcs w/ a particular kubespray hash is sufficient to advance the version w/o updating the hash in the script.

Reply:
@JohnStrunk https://github.com/gluster/gcs/pull/105 PTAL